students/ShinTran/lesson04/kata.py

```diff
@@ -0,0 +1,78 @@
+'''
+Shin Tran
+Python 210
+Lesson 4 Assignment
+'''
+
+
+import string # For punctuation
+import random # For a random number generator
+
+# To store all the two word combo as keys and the following word as a value
+word_dict = {}
+
+
+# Opens a text file, parses though it line by line, storing two consecutive
+# words as keys, and the following word as a value
+def open_file():
+	global word_dict
+	sentence = []
+	contents = open('sherlock.txt')
```

Comment:
it is preferred to use `with` statement when working with files. (refer to course materials)